packages/rest/src/rest.component.ts

```diff
@@ -89,7 +89,10 @@ export class RestComponent implements Component {
   ) {
     app.bind(RestBindings.SEQUENCE).toClass(DefaultSequence);
     const apiSpec = createEmptyApiSpec();
-    // Merge the OpenAPI `servers` spec from the config into the empty one
+    // Merge the OpenAPI `info` & `servers` spec from the config into the empty one
+    if (config && config.openApiSpec && config.openApiSpec.info) {
```

Comment:
Instead of copying just the info object, maybe it makes sense to pass a partial schemaobject for initialisation?

Reply:
Yes, the whole object could also be passed

Reply:
should I delete `openapi: {{version}}`?